public/app/plugins/datasource/jaeger/components/QueryEditor.tsx

```diff
@@ -32,16 +36,12 @@ export function QueryEditor({ datasource, query, onChange, onRunQuery }: Props)
         return (
           <InlineFieldRow>
             <InlineField label="Trace ID" labelWidth={21} grow>
-              <Input
-                aria-label={selectors.components.DataSource.Jaeger.traceIDInput}
```

Comment:
I think this aria-label is used by [a e2e test](https://github.com/grafana/grafana/blob/main/e2e/suite1/specs/trace-view-scrolling.spec.ts). You can try changing the test to something like we have in Prometheus (I haven't tested it though):

```diff
- e2e.components.DataSource.Jaeger.traceIDInput().should('be.visible').type('long-trace');
+ e2e.components.QueryField.container().should('be.visible').type('long-trace');
```